addons/upgrade/to-13.0-convert-arubaswitch-types.pl

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/perl
+
+=head1 NAME
+
+to-13.0-convert-switch-types.pl
+
+=head1 DESCRIPTION
+
+Convert some switch types and use Switch OS versions
+
+=cut
+
+use strict;
+use warnings;
+use lib qw(/usr/local/pf/lib);
+use lib qw(/usr/local/pf/lib_perl/lib/perl5);
+use pf::IniFiles;
+use NetAddr::IP;
+use pf::util qw(valid_ip valid_mac run_as_pf);
+use pf::file_paths qw(
+    $switches_config_file
+);
+use File::Copy;
+
+run_as_pf();
+
+my $file = $switches_config_file;
+
+if (@ARGV) {
+    $file = $ARGV[0];
+}
+
+our %types = (
+    'Cisco::Catalyst_2950' => 'Cisco::Cisco_IOS_12_x',
```

Comment:
Cisco ?!

Reply:
I created this for Aruba and because @Ieno77 is working on Aruba I left it for him to make the final adjustments.